### ISSUE TITLE
XGBoost tree vs usual gradient boosting tree

### ISSUE THREAD

**Issue by sergun** [dmlc/xgboost#4391]
Hi!

Is training XGBoost tree on each iteration (forget about regularisation and other usuful features) the same as training regression tree in usual gradient boosting with samples x_i with weights h_i and targets g_i/h_i?

It is easy to see that leaf weights are calculated this way.
But it seems to me that Gain in XGBoots is not variance based as in normal regression tree for RMSE.

If there is difference here which algorithm (simple GB or XGBoost) gives better tree structure and why?

@tqchen any comments from author of this great library?

Thanks a lot!
Sergey.

**Comment by hcho3** [comment]
Take a look at the XGBoost paper: https://arxiv.org/pdf/1603.02754v2.pdf

XGBoost uses second-order gradients to fit trees.

**Comment by tqchen** [comment]
It is a topic that fits better at https://discuss.xgboost.ai/
